README.md

```diff
@@ -1,193 +1,198 @@
+1. From the panel drop-down, click Explore
+1. In the Explore view, go to the data source drop-down and select Loki
+1. Expand one of the loglines and click the blue "Tempo" button next to the `TraceID` to see trace information
 
-The following instructions will help you go from metrics to logs to traces.
+### Go Metrics -> Traces -> Logs
 
-1. In Grafana, go to the Explore view.
-2. From the data source drop-down and select Prometheus.
-3. Run the following query.
+1. Navigate to [http://localhost:8080/](http://localhost:8080) and click the link for Grafana
+1. In Grafana, go to the Explore view
+1. From the data source drop-down and select `prometheus-exemplars`.
+1. Run the following query.
 `histogram_quantile(.99, sum(rate(tns_request_duration_seconds_bucket{}[1m])) by (le))`
-4. Click on an exemplar data.
-5. Click on the log icon on a span line to view the log details.
+1. Hover over one of the exemplar data points
+1. Click the blue "Query with Tempo" button next to the `TraceID` to see trace information
+1. Click the log icon on a span line to view log details
 
 ### Explore LogQL V2
 
-1. In Grafana, go to the Explore view.
-2. From the data source drop-down and select Loki.
-3. Run the following query.
+1. Navigate to [http://localhost:8080/](http://localhost:8080) and click the link for Grafana
+1. In Grafana, go to the Explore view
+1. From the data source drop-down and select Loki
+1. Run the following query.
  `{job="tns/app"} | logfmt | level="info" | status>=500 and status <=599 and duration > 50ms`
-4. Choose a logline with a traceID of Tempo.
+1. Expand one of the loglines and click the blue "Tempo" button next to the `TraceID` to see trace information
 
-## Disable TNS cluster
+## Cleaning up (or starting over)
 
-To disable your cluster, run this command:
+### Redeploying the Kubernetes resources
+
+```sh
+$ rm -rf tanka/
+$ ./install
+```
+
+### Managing the `k3d` cluster
+
+To stop your cluster, run this command:
 
 ```sh
 $ k3d cluster stop tns
 ```
 
-To re-enable the cluster, run this command:
+To re-start the cluster, run this command:
 
 ```sh
 $ k3d cluster start tns
 ```
 
-## Remove TNS cluster
-
-To remove your cluster, run this command:
+To delete your cluster, run this command:
 
 ```sh
 $ k3d cluster delete tns
-$ rm -rf tanka
 ```
 
+To recreate the cluster, delete it as above, then re-run the steps in the [Kubernetes Cluster](#kubernetes-cluster) section.
+
 ## Troubleshooting
 
-**Issue:** 404 error when trying to load Tempo traces.
-   
-**Solution:** This is likely because the jaeger agent is not running correctly. Check that all pods were successfully scheduled.
+### 404 error when trying to load Tempo traces.
+
+This is likely because the jaeger agent is not running correctly. Check that all pods were successfully scheduled.
+
+### `permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock` error when creating `k3d` cluster
```

Comment:
The heading should be reworded.

Reply:
Updated to "Permission error when creating `k3d` cluster" in https://github.com/grafana/tns/pull/36/commits/ce7c172a90d7431c477aa3b65e4b5c1eb8e722b8